vms/platformvm/service.go

```diff
@@ -544,15 +544,26 @@ func (s *Service) GetSubnets(_ *http.Request, args *GetSubnetsArgs, response *Ge
 				continue
 			}
 
-			unsignedTx := subnet.Unsigned.(*txs.CreateSubnetTx)
-			owner := unsignedTx.Owner.(*secp256k1fx.OutputOwners)
-			controlAddrs := []string{}
-			for _, controlKeyID := range owner.Addrs {
+			subnetOwner, err := s.vm.state.GetSubnetOwner(subnetID)
+			if err == database.ErrNotFound {
```

Comment:
That would leave "holes" in the "response.Subnets". Was that your intent ?

Reply:
We shouldn't be leaving "holes" here since we're returning all the subnets. Removed the `database.ErrNotFound` check, good catch!